drivers/adc/CMakeLists.txt

```diff
@@ -4,76 +4,79 @@ zephyr_syscall_header(${ZEPHYR_BASE}/include/zephyr/drivers/adc.h)
-zephyr_library_sources_ifdef(CONFIG_ADC_SAM_AFEC	adc_sam_afec.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_NRFX_ADC	adc_nrfx_adc.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_NRFX_SAADC	adc_nrfx_saadc.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_SAM		adc_sam.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_SAM0		adc_sam0.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_STM32		adc_stm32.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_STM32WB0	adc_stm32wb0.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_XEC		adc_mchp_xec.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_LMP90XXX	adc_lmp90xxx.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_MCP320X		adc_mcp320x.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_MCP356XR	adc_mcp356xr.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_NPCX		adc_npcx.c)
-zephyr_library_sources_ifdef(CONFIG_USERSPACE		adc_handlers.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_CC32XX		adc_cc32xx.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_CC13XX_CC26XX		adc_cc13xx_cc26xx.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_CC23X0		adc_cc23x0.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_EMUL		adc_emul.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_TEST		adc_test.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ADS1X1X		adc_ads1x1x.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_GD32		adc_gd32.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ADS1112		adc_ads1112.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ADS1119		adc_ads1119.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ADS7052		adc_ads7052.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ADS1X4S0X		adc_ads1x4s0x.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ADS131M02		adc_ads131m02.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_RPI_PICO	adc_rpi_pico.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_XMC4XXX		adc_xmc4xxx.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ESP32		adc_esp32.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ESP32_DMA	adc_esp32_dma.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_GECKO_ADC   	adc_gecko.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_SILABS_IADC   	adc_silabs_iadc.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_SILABS_SIWX91X 	adc_silabs_siwx91x.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_INFINEON_CAT1	adc_ifx_cat1.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_INFINEON_HPPASS_SAR	adc_ifx_hppass_sar.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_INFINEON_AUTANALOG_SAR	adc_ifx_autanalog_sar.c)
+zephyr_library_sources_ifdef(CONFIG_ADC adc_common.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_SHELL adc_shell.c)
+zephyr_library_sources_ifdef(CONFIG_USERSPACE adc_handlers.c)
+
+# zephyr-keep-sorted-start
+zephyr_library_sources_ifdef(CONFIG_ADC_AD405X adc_ad405x.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_AD4114 adc_ad4114.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_AD4130 adc_ad4130.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_AD4170 adc_ad4170.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_AD559X adc_ad559x.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_AD7124 adc_ad7124.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_ADS1112 adc_ads1112.c)
```

Comment:
I know the current 2nd column alignment was broken due to varying length on different lines.

However, I personally find this hard to read, and aligning files on second column does help readability, but the risk is of course that all lines are changed if a new and longer `CONFIG_x` var is added.

Should this be discussed further or is there consensus regarding this ?

Note, only placing this comment here, though we have more occurrences.